umap/templatetags/umap_tags.py

```diff
@@ -25,7 +26,7 @@ def map_fragment(map_instance, **kwargs):
     page = kwargs.pop("page", None) or ""
     unique_id = prefix + str(page) + "_" + str(map_instance.pk)
     return {
-        "map_settings": json.dumps(map_settings),
+        "map_settings": json.dumps(map_settings, cls=DjangoJSONEncoder),
```

Comment:
I believe it's a matter of taste :-) I prefer to rely on `DjangoJSONEncoder` rather than doing the typing changes myself every so often.

My hope is that it probably will help solve other problems down the line.

Reply:
+1 for using the `DjangoJSONEncoder`, it will take care of [other types](https://docs.djangoproject.com/en/5.0/topics/serialization/#djangojsonencoder) for us which are so easy to miss.

Reply:
Then maybe having a `json_dumps` util ? Importing and using the encoder every where does not seem very DRY